src/test/java/io/openliberty/tools/intellij/it/GradleSingleModMPProjectTest.java

```diff
@@ -88,6 +96,22 @@ public String getWLPInstallPath() {
         return WLP_INSTALL_PATH;
     }
 
+    /**
+     * This method expands specific nodes within a project tree represented by a JTreeFixture.
+     * It locates the ProjectFrameFixture using a remote robot and retrieves a JTreeFixture for the project view.
+     * Then, it expands the nodes corresponding to the given path within the project tree.
+     *
+     * @return JTreeFixture representing the expanded project tree nodes.
+     */
+    @Override
+    public JTreeFixture expandProjectTreeNodes() {
+        // get a JTreeFixture reference to the file project viewer entry
+        ProjectFrameFixture projectFrame = remoteRobot.find(ProjectFrameFixture.class, Duration.ofMinutes(2));
+        JTreeFixture projTree = projectFrame.getProjectViewJTree(getSmMPProjectName());
+        UIBotTestUtils.openFile(remoteRobot, getSmMPProjectName(), "server.env", getSmMPProjectName(), "build", "wlp", "usr", "servers", "defaultServer");
```

Comment:
Can we use this method to expand any node in the project tree by passing file name and path.

Reply:
Removed the newly created method. Used UIBotTestUtils.openFile to open the server.env file by specifying paths seperately for maven and gradle files.